endpoints/openrtb2/video_auction_test.go

```diff
@@ -53,8 +53,8 @@ func TestVideoEndpointImpressionsNumber(t *testing.T) {
 	}
 
 	assert.Len(t, ex.lastRequest.Imp, 11, "Incorrect number of impressions in request")
-	assert.Equal(t, string(ex.lastRequest.Site.Page), "prebid.com", "Incorrect site page in request")
-	assert.Equal(t, ex.lastRequest.Site.Content.Series, "TvName", "Incorrect site content series in request")
+	assert.Equal(t, "prebid.com", string(ex.lastRequest.Site.Page), "Incorrect site page in request")
+	assert.Equal(t, "TvName", ex.lastRequest.Site.Content.Series, "Incorrect site content series in request")
```

Comment:
Many assertions in the `video_auction_test.go` were reversed. The expected value should be provided before the actual. This produced confusing failures while i was upgrading the OpenRTB library. Fixed while I was working in the file.